### PR TITLE
WIP: Fix Failing Tests

### PR DESCRIPTION
**Pull request by thestephenmarshall**

### CODE REVIEW

File: .github/workflows/ruby-ci.yml [hunk 1]
```diff
@@ -55,12 +55,13 @@ jobs:
 
     - name: Install Selenium
       run: |
-          sudo gem install selenium-webdriver
+          gem install selenium-webdriver
```
Comment (thestephenmarshall) [inline .github/workflows/ruby-ci.yml:58]: Not sure why this lib isn't able to find `chrome-driver` any ideas @wadewinningham ?

Comment (wadewinningham) [reply .github/workflows/ruby-ci.yml:58]: I see this:

```
Selenium::WebDriver::Error::UnknownError: unknown error: Chrome failed to start: crashed.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
```

Which makes it appear to me like it's there, but fails to start.

This was the main, relevant, item I came across while googling. https://github.com/actions/virtual-environments/issues/506 has some stuff to try in the CI setup which seems relevant.

Comment (thestephenmarshall) [reply .github/workflows/ruby-ci.yml:58]: 😬 I've tried all the things from that issue. Still no dice..

Comment (wadewinningham) [reply .github/workflows/ruby-ci.yml:58]: If I were to guess, that's due to an out of memory error, but not sure how to confirm that being the case or not.

Comment (wadewinningham) [reply .github/workflows/ruby-ci.yml:58]: In https://github.com/SeleniumHQ/selenium/issues/4961, there's a suggestion to run chrome in `--no-sandbox` mode.

**Pull request merged**